CRMLite.TransactionStoreAPI/Startup.cs

```diff
@@ -20,8 +23,14 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            var connectionString = Configuration.GetConnectionString("Default");
+            DbConnection connection = new SqlConnection(connectionString);
+
```

Comment:
For example
DbConnection connection = new SqlConnection(Configuration.GetConnectionString("Default"));